libass/libass.def

```diff
@@ -0,0 +1,47 @@
+EXPORTS
+ass_library_version
+ass_library_init
```

Comment:
This file is identical to the already existing `libass/libass.sym` only with an added `EXPORTS` as the first line. Rather than mainting the same info twice, the def-file should be auto-generated by meson from the sym-file (and the def-file added to `.gitignore`).

Reply:
Will do.